Sources/AppcuesKit/Data/Networking/DecodingExperienceConfig.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DecodingExperienceConfig.swift
+//  AppcuesKit
+//
+//  Created by Matt on 2022-12-12.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import Foundation
+
+/// Object that stores the configuration options provided in ``Experience.Trait`` and ``Experience.Action`` models.
+public class DecodingExperienceConfig: NSObject {
```

Comment:
checking understanding - is this basically just to provide a convenience wrapper type with helpers/extensions rather than having everything work with `[String: KeyedDecodingContainer<JSONCodingKeys>]` directly? It could be done just using type type or a `typedef` perhaps? Granted, this probably simplifies things.

Reply:
re: naming, given some other chat, maybe `ExperiencePluginConfig`? `AppcuesPluginConfig`. Nothing really jumps out to me.